README.md

```diff
@@ -7,24 +7,24 @@ We built a portable soil moisture tracker to help monitor plant health by lettin
 
 ![final](public/images/final.png)
 
-## Technologies 
+## Technologies
 
-H A R D W A R E 
+H A R D W A R E
 
-* Raspberry Pi 4 
+* Raspberry Pi 4
 * Adafruit Soil Moisture Sensor
 * 4 Pin JST Cable
 * LCD touchscreen
 * Jumper wires
-* Battery pack 
+* Battery pack
```

Comment:
I think it might be a good idea to have a diagram of how the screen and the sensor were attached to the same pins, or a link to the blog that describes the same